### PR TITLE
Simple get single user authorized implementation

### PR DESCRIPTION
**Pull request by sergiitk**
- Implements `NorthstarClient` options configuration
- Implements `NorthstarUser` and `NorthstarUserAuthorized`
- Introduces authorized to the NorthStar API
- Includes [`dotenv`](https://github.com/motdotla/dotenv) to set env data for testing
- Test everything!
- Bump version to `0.0.2`.

### CODE REVIEW

File: lib/northstar-user.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  // Construct from JSON.
+  constructor(data) {
+    // Computed.
+    this.isAuthorized = false;
+
+    // TODO: object id?
+    this.id = data.id;
+
+    // String.
+    this.firstName = data.first_name;
```
Comment (deezone) [inline lib/northstar-user.js:17]: Do we want to format the data regardless of what Northstar returns. Capital first letter for first and last name?

File: test/northstar-client.test.js [hunk 2]
```diff
@@ -3,60 +3,89 @@
...
       // Just ensure presence.
-      user.should.have.properties(['updated_at', 'created_at']);
+      user.should.have.properties(['updatedAt', 'createdAt']);
     }
 
+    // Constructor.
+    describe('constructor', () => {
+      // Test new instance.
+      it('without options should throw a TypeError', () => {
+        (() => new NorthstarClient()).should.throw(TypeError);
```
Comment (deezone) [inline test/northstar-client.test.js:62]: Perhaps a `getNorthstarClient()` just like `getUnauthorizedClient()`?

Comment (sergiitk) [inline test/northstar-client.test.js:62]: No. Functions `getUnauthorizedClient()` and `getAuthorizedClient()` are helpers to construct  `NorthstarClient()` with options (`baseURI` and `apiKey`). In this case I'm building `NorthstarClient()` incorrectly, without options, to ensure throws `TypeError`.

File: lib/northstar-user.js (hunk 1 above)
Comment (sergiitk) [inline lib/northstar-user.js:17]: Sorry?

Comment (deezone) [inline lib/northstar-user.js:17]: Northstar returns:
- `bobby joe`
  constructor(data) sets:

```
this.firstName = data.first_name.replace(/\w\S*/g, 
  (tStr)-> { 
    return tStr.charAt(0).toUpperCase() + tStr.substr(1).toLowerCase(); 
  }
);
```
- now it sets this.firstName to `Bobby Joe`.

**Note**:
I'm sure I made typos in the code above. Just trying to get my idea across.

Comment (sergiitk) [inline lib/northstar-user.js:17]: But that's not a full name, it's just a `firstName`.

Comment (sergiitk) [inline lib/northstar-user.js:17]: I'd suggest to add a computed getter, for example `getFirstNameCapitalized()`.

**Comment by deezone** [comment]
👍

**Pull request merged**